### DISCUSSION TITLE
MoBi building blocks not appearing

### DISCUSSION THREAD

**Discussion by Sarang977** [Open-Systems-Pharmacology/Forum#1852] [Questions & Problems] [answered]
Suddenly MoBi stopped working properly. Building blocks and saved models appear as blank screens upon opening. I have restarted the PC and reinstalled the software but the problem persists. However, PK-sim is working fine.

Any suggestions to resolve this issue?

![image](https://github.com/user-attachments/assets/6bdd317e-0946-4f9e-bbe2-ebf51817c777)

Thanks

**Comment by Yuri05** [comment, marked as answer]
s. here https://github.com/Open-Systems-Pharmacology/Forum/discussions/1150

**Reply by Sarang977** [reply]
Thanks
